cmd/notary/tuf.go

```diff
@@ -368,19 +365,13 @@ func getTransport(gun string, readOnly bool) http.RoundTripper {
 		if !filepath.IsAbs(rootCAFile) {
 			rootCAFile = filepath.Join(configPath, rootCAFile)
 		}
-		rootCert, err := trustmanager.LoadCertFromFile(rootCAFile)
-		if err != nil {
-			fatalf("could not load root ca file. %s", err.Error())
-		}
-		rootPool.AddCert(rootCert)
 	}
 
-	// skipTLSVerify is false by default so verification will
-	// be performed.
-	tlsConfig := &tls.Config{
-		InsecureSkipVerify: mainViper.GetBool("remote_server.skipTLSVerify"),
-		MinVersion:         tls.VersionTLS10,
-		RootCAs:            rootPool,
+	tlsConfig, err := utils.ConfigureClientTLS(
+		rootCAFile, "", mainViper.GetBool("remote_server.skipTLSVerify"),
```

Comment:
You're right, I was just relying on the viper documentation saying that `Get*` defaults to the zero value.  I'll do an explicit check, thanks!